instrumentation/github.com/aws/aws-lambda-go/otellambda/xrayconfig/README.md

```diff
@@ -0,0 +1,77 @@
+# Recommended Configurations for OpenTelemetry AWS Lambda Instrumentation with AWS X-Ray
+
+[![Go Reference][goref-image]][goref-url]
+[![Apache License][license-image]][license-url]
+
+This module provides recommended configuration options for [`AWS Lambda Instrumentaton`](https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/instrumentation/github.com/aws/aws-lambda-go/otellambda) when using [AWS X-Ray](https://aws.amazon.com/xray/).
+
+## Installation
+
+```bash
+go get -u go.opentelemetry.io/contrib/instrumentation/github.com/aws/aws-lambda-go/otellambda/xrayconfig
+```
+
+## Usage
+
+Create a sample Lambda Go application instrumented by the `otellambda` package such as below.
+
+```go
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/aws/aws-lambda-go/lambda"
+	"go.opentelemetry.io/contrib/instrumentation/github.com/aws/aws-lambda-go/otellambda"
+)
+
+type MyEvent struct {
+	Name string `json:"name"`
+}
+
+func HandleRequest(ctx context.Context, name MyEvent) (string, error) {
+	return fmt.Sprintf("Hello %s!", name.Name ), nil
+}
+
+func main() {
+	lambda.Start(otellambda.WrapHandlerFunction(HandleRequest))
+}
+```
+
+Now configure the instrumentation with the provided options to export traces to AWS X-Ray via [the OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) running as a Lambda Extension. Instructions for running the OTel Collector as a Lambda Extension can be found in the [AWS OpenTelemetry Documentation](https://aws-otel.github.io/docs/getting-started/lambda).
```

Comment:
In addition to uploading traces to X-Ray, does this library also
* extract trace context from `X-Amzn-Trace-Id` headers on inbound requests
* apply an `X-Amzn-Trace-Id` header to outbound requests
It may be worth documenting clearly in the README.